gamemodes/hunters_glee/gamemode/sh_shopitems.lua

```diff
@@ -2292,6 +2292,20 @@ local function ar2Purchase( purchaser )
 
 end
 
+local function fragPurchase( purchaser )
+    local frag = purchaser:GetWeapon( "weapon_frag" )
+    if IsValid( frag ) then
+        purchaser:GiveAmmo( 3,    "Grenades",         false )
+
+    else
+        purchaser:GiveAmmo( 4,    "Grenades",         false )
```

Comment:
```suggestion
        purchaser:GiveAmmo( 7,    "Grenades",         false )

    else
        purchaser:GiveAmmo( 8,    "Grenades",         false )
```
Make it give more grenades, so that buying the item is a hard choice instead of a top-up